src/containers/PageBuilder/Primitives/List/List.module.css

```diff
@@ -14,9 +14,16 @@
       list-style-type: square;
     }
   }
+
+  & + * {
+    margin-top: 24px;
+  }
 }
 
 .ol {
+  & + * {
+    margin-top: 24px;
+  }
```

Comment:
What does this mean?
A list (ul & ol) should not be used unless there's list-item (li) inside.
Did you find a bug?